Core/test/Test_AuthManager.py

```diff
@@ -45,6 +45,11 @@
     {
       DN = /User/test/DN/CN=userB
     }
+    userS
```

Comment:
You may also add a test case for a user who is suspended in a different VO, but not the one it is presenting ?

Reply:
Yes, this can be added

Reply:
Done